tests/HTTP/SapiTest.php

```diff
@@ -117,4 +117,49 @@ function testSend() {
 
     }
 
+    /**
+     * @runInSeparateProcess
+     */
+    function testSendLimitedByContentLengthString() {
+
+        $response = new Response(200);
+
+        $response->addHeader('Content-Length', 19);
+        $response->setBody('Send this sentence. Ignore this one.');
+
+        ob_start();
+
+        Sapi::sendResponse($response);
+
+        $result = ob_get_clean();
+        header_remove();
```

Comment:
Oh, I see. However, even if it did, there is the ```@runInSeparateProcess``` tag, so it is isolated. Also, testSend depends on the purity of the headers too, so it shouldn't be a problem, I think.